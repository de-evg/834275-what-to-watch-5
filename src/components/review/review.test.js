import React from "react";
import renderer from "react-test-renderer";
import Review from "./review";

const movieReview = {
  text: `Some text`,
  date: new Date(),
  author: `Author Name`,
  userRating: `10`
};

it(`Should Review render correctly`, () => {
  const tree = renderer
    .create(<Review
      movieReview={movieReview}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
