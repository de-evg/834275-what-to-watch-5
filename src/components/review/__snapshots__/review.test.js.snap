// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Review render correctly 1`] = `
<div
  className="review"
>
  <blockquote
    className="review__quote"
  >
    <p
      className="review__text"
    >
      Some text
    </p>
    <footer
      className="review__details"
    >
      <cite
        className="review__author"
      >
        Author Name
      </cite>
      <time
        className="review__date"
        dateTime="2020-10-19"
      >
        November 19, 2020
      </time>
    </footer>
  </blockquote>
  <div
    className="review__rating"
  >
    10
  </div>
</div>
`;
